app/web/features/messages/groupchats/GroupChatView.tsx

```diff
@@ -197,101 +231,139 @@ export default function GroupChatView({ chatId }: { chatId: number }) {
             </HeaderButton>
 
             {groupChat?.isDm ? (
-              <Link
-                href={`/user/${getDmUsername(
-                  groupChatMembersQuery,
-                  currentUserId
-                )}`}
-              >
-                <a>
-                  <PageTitle className={classes.title}>
-                    {title || <Skeleton width={100} />}
-                  </PageTitle>
-                </a>
-              </Link>
+              <div className={classes.title}>
+                <Link
```

Comment:
I don't think the same margin in group chats as they do in DMs
![Screenshot 2022-02-09 at 01 06 01](https://user-images.githubusercontent.com/13669362/153102482-c7216f50-65c2-4fe0-802a-0e237f7c051b.png)